content/kubermatic/main/architecture/enterprise-edition/_index.md

```diff
@@ -19,6 +19,7 @@ Kubermatic Kubernetes Platform (KKP) is an open-source product, and both the Com
 | Resource Quotas                             | KKP admins can define consumption caps for different projects.                                                                                                                                |
 | Group Project bindings                      | Projects can be bound to user groups defined in your authentication provider.                                                                                                                 |
 | Edge capabilities                           | Independent operation from the control plane and workload allocation among different clusters.                                                                                                |
+| Application catalog                         | Easy solution for deploying and managing the most popular applications in your Kubernetes User Clusters right from KKP.                                                                       |
```

Comment:
Nit: Other entries in this table spell user clusters lowercase, so I think this should be consistent:

```suggestion
| Application catalog                         | Easy solution for deploying and managing the most popular applications in your Kubernetes user clusters right from KKP.                                                                       |
```

Same goes for the v2.24 version of course.